### PR TITLE
[Translation] Translate Resistor Color exercise

### PR DESCRIPTION
**Pull request by Bscruz19**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -816,6 +816,16 @@
...
       "uuid": "7ed99829-acd8-4724-b92f-85146f1709f9"
+    },
+    {
+      "core": false,
+      "difficulty": 1,
+      "slug": "resistor-color",
+      "topics": [
+        "enumerables"
+      ],
+      "unlocked_by": null,
```
Comment (neenjaw) [inline config.json]: pick a core problem to unlock this.

File: exercises/resistor-color/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+Mnemonics map the colors to the numbers, that, when stored as an array, happen to map to their index in the array: Better Be Right Or Your Great Big Values Go Wrong.
+
+
+## Running tests
+
+Execute the tests with:
+
+```bash
+$ elixir resistor_color_test.exs
```
Comment (neenjaw) [inline exercises/resistor-color/README.md]: please update to the new mix project style

File: exercises/resistor-color/example.exs [hunk 3]
```diff
@@ -0,0 +1,11 @@
+defmodule ResistorColor do
+  @moduledoc false
+
+  @colors ["black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"]
+
+  @spec colors() :: List.t()
```
Comment (neenjaw) [inline exercises/resistor-color/example.exs]: need to specify the `@opaque` type

File: exercises/resistor-color/resistor_color_test.exs [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  Code.load_file("resistor_color.exs", __DIR__)
+end
+
+ExUnit.start()
+ExUnit.configure(exclude: :pending, trace: true)
+
+defmodule ResistorColorTest do
+  use ExUnit.Case
+
+  @colors ["black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"]
```
Comment (neenjaw) [inline exercises/resistor-color/resistor_color_test.exs]: move this to a local variable inside of the `returns all colors` test

File: exercises/resistor-color/resistor_color_test.exs [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+  @tag :pending
+  test "returns orange color code" do
+    assert ResistorColor.code("orange") == 3
+  end
+
+  @tag :pending
+  test "returns violet color code" do
+    assert ResistorColor.code("violet") == 7
+  end
```
Comment (neenjaw) [inline exercises/resistor-color/resistor_color_test.exs]: write a test for each color

File: exercises/resistor-color/resistor_color_test.exs [hunk 6]
```diff
@@ -0,0 +1,37 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("resistor_color.exs", __DIR__)
+end
```
Comment (neenjaw) [inline exercises/resistor-color/resistor_color_test.exs]:
```suggesstion
```

no longer neccessary in the new mix project style

File: exercises/resistor-color/resistor_color_test.exs [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+ExUnit.start()
+ExUnit.configure(exclude: :pending, trace: true)
```
Comment (neenjaw) [inline exercises/resistor-color/resistor_color_test.exs]: move to `test/test_helper.exs` in the mix project

**Comment by neenjaw** [comment]
squash and rebase your commits once we've worked through all the issues, until then it makes the review process less clear (learned this the hard way myself).

File: exercises/resistor-color/README.md (hunk 2 above)
Comment (Bscruz19) [reply exercises/resistor-color/README.md]: @neenjaw done!

File: exercises/resistor-color/resistor_color_test.exs (hunk 4 above)
Comment (Bscruz19) [reply exercises/resistor-color/resistor_color_test.exs]: @neenjaw done!

File: exercises/resistor-color/resistor_color_test.exs (hunk 5 above)
Comment (Bscruz19) [reply exercises/resistor-color/resistor_color_test.exs]: @neenjaw done!

File: exercises/resistor-color/resistor_color_test.exs (hunk 6 above)
Comment (Bscruz19) [reply exercises/resistor-color/resistor_color_test.exs]: @neenjaw removed!

File: exercises/resistor-color/resistor_color_test.exs (hunk 7 above)
Comment (Bscruz19) [reply exercises/resistor-color/resistor_color_test.exs]: @neenjaw moved!

File: exercises/resistor-color/lib/resistor_color.ex [hunk 8]
```diff
@@ -0,0 +1,11 @@
+defmodule ResistorColor do
+  @moduledoc false
+
+  @spec colors() :: list(String.t())
+  def colors do
+  end
+
+  @spec code(String.t()) :: integer
```
Comment (neenjaw) [inline exercises/resistor-color/lib/resistor_color.ex]: Should be integer() to follow the style

Comment (Bscruz19) [reply exercises/resistor-color/lib/resistor_color.ex]: @neenjaw done!

File: exercises/resistor-color/example.exs (hunk 3 above)
Comment (Bscruz19) [reply exercises/resistor-color/example.exs]: @neenjaw done!

**Comment by neenjaw** [comment]
Your build is failing because example.exs needs to be mix formatted

**Comment by neenjaw** [comment]
Looks good to me, I'd like one of the other maintainers to look it over as well, then should be good to go!  Thanks again!

**Comment by neenjaw** [comment]
Ready @Bscruz19?

**Pull request merged**